apps/web/components/sections/project/update/update-tab-section.tsx

```
@@ -167,7 +167,7 @@ export function ProjectUpdatesTabSection() {
 			fetchUpdates()
 		}
 		// eslint-disable-next-line react-hooks/exhaustive-deps -- fetchUpdates is stable; adding it causes unnecessary refetches
-	}, [page, projectId, isCreatingUpdate])
+	}, [isCreatingUpdate, fetchUpdates])
```

Biome (2.4.13)

[error] fetchUpdates changes on every re-render and should not be used as a hook dependency.

(lint/correctness/useExhaustiveDependencies)